code/modules/mob/living/simple_animal/bot/SuperBeepsky.dm

```diff
@@ -0,0 +1,180 @@
+/mob/living/simple_animal/bot/secbot/grievous //This bot is powerful. If you managed to get 4 eswords somehow, you deserve this horror. Emag him for best results.
+	name = "General Beepsky"
+	desc = "Is that a secbot with four eswords in its arms...?"
+	icon = 'icons/mob/aibots.dmi'
+	icon_state = "grievous0"
+	health = 150
+	maxHealth = 150
+	baton_type = /obj/item/melee/transforming/energy/sword
+	var/obj/item/melee/transforming/energy/sword/weapon = new
+	var/block_chance = 50
```

Comment:
Move this below base speed please. New vars should be kept at the bottom just for readability sake.